src/ctap/mod.rs

```diff
@@ -717,6 +717,22 @@ impl CtapState {
         Ok(())
     }
 
+    fn check_cred_protect(
```

Comment:
Maybe we can rename this function, to make its purpose clear. This policy check is only correct if performed for a credential coming from an allow list or exclude list. There is also the `is_discoverable` function that does the equivalent check for non-allow-listed credentials (rk=true) during GetAssertion. `check_listed_...` or something similar?

Reply:
done